qa/suites/rgw/lua/supported-all-distro$/centos_latest.yaml

```diff
@@ -0,0 +1 @@
+../.qa/distros/supported/centos_latest.yaml
```

Comment:
please symlink to `supported-all-distro$` so it runs against ubuntus also

Reply:
> please symlink to `supported-all-distro$` so it runs against ubuntus also

don't think it can run against ubuntu, since i have these lines:
https://github.com/ceph/ceph/pull/52931/files#diff-bcce586a3089246b6b25f464579488d340cb2f206593d24c7ac420b99a98670fR29

Reply:
can you not use extra_system_packages to do that in yaml instead?

Reply:
are you sure you need to? i thought ceph dependencies already required that

regardless, if you need special logic for rpm distros, you can add logic for deb too

Reply:
locally i had to enable crb for lua-devel on centos9. will try in teuthology without that
debian does not need special repo enabled

Reply:
maybe that stuff should be in the bootstrap script